packages/node-resolve/src/package/resolvePackageImports.js

```diff
@@ -0,0 +1,49 @@
+import path from 'path';
+import fs from 'fs';
+import { pathToFileURL } from 'url';
+
+import { createBaseErrorMsg, InvalidModuleSpecifierError } from './utils';
+import resolvePackageImportsExports from './resolvePackageImportsExports';
+
+function findPackageJson(base) {
+  const { root } = path.parse(base);
+  let current = base;
+
+  while (current !== root && !current.endsWith('node_modules')) {
```

Comment:
I think the name of `node_modules` is configurable on an option?

Reply:
Good point, made this configurable.